config/locales/api/en.yml

```diff
@@ -11,6 +11,7 @@ en:
       invalid_otp: 'Your OTP does not match. Try again?'
       expired_otp: 'You need a fresh OTP. Request a new one.'
       invalid_password: 'Your password does not match. Try again?'
+      account_locked: "Your account is locked out for %{minutes} minutes. Please wait and try again later."
```

Comment:
Have we run the messaging by design 👀 ?

Reply:
Thanks for asking. Am suggesting a slightly tighter language. Also, can we update the expired OTP one too?

Your OTP is expired. Request a new one. 

Your account has been locked for the next X minutes. Please wait and try again.

Reply:
Thanks, updated!